blockchain-interface/src/error.rs

```diff
@@ -51,7 +51,7 @@ pub enum BlockchainError {
     #[error(
         "A full genesis config is required for initializing a history node on \
         the mainnet for the first time. Obtain the full genesis config (it's \
-        not in the repository) and set the `NIMIQ_OVERRIDE_MAINNET_CONFIG` \
+        not in the repository) and set the `NIMIQ_OVERRIDE_MAINET_CONFIG` \
```

Comment:
I think it should rather be the environment variable that should get fixed, optionally with a fallback for the typo.

Reply:
Why? Now with the v1 release version we have the chance to make it right also in the nimiq-client.

Reply:
I agree. Now with the 1.0 release we have the chance to make it right.

Reply:
Done